### PR TITLE
[AFTERSHOCK] House Palette + New AFS house

### PR DESCRIPTION
**Pull request by Mom-Bun**
#### Summary
Mods "Add a single AFS house"

#### Purpose of change

Personally, it'll be a bit boring if the cities are just formless ruins, so having more uncommon and rarer buildings like houses would be nice, and to properly make these we need well... palettes so we can actually make it? 

This should add an extremely basic palette for houses that were able to withstand the test of time on Salus IV

#### Describe the solution

Added a new palette as well as a house.

A few notes, I didn't want to make this PR too big or anything, so I tried to stay small, so this doesn't add any new itemgroups, furniture, terrain, or items, so it is REALLY basic for now.

#### Describe alternatives you've considered

Not adding in the item groups yet, I figured it'll make it easier to just cut and paste in when we finally get around to replacing them instead of having to put in new item groups once we eventually get to it.

#### Testing

Game boots up correctly, and houses spawn correctly with loot + roof.

#### Additional context

The fridge and cupboards still spawn vanilla food, so this makes food easy to get, this'll need replacing with our own AFS food group so we can spawn inedible food, and rarer. 

The bookshelves spawn nothing, I figured it'll be best to wait until we fill out our books in AFS before adding them, or at least a book item group.

All the furniture is still wooden and everything, this'll require replacement with plastic AFS variants as well.

### CODE REVIEW

File: data/mods/Aftershock/maps/mapgen/houses/houses.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        "   oI.J........H..2..y+ ",
+        "   +P.S........H.....y+ ",
+        "   +M^M...E.......Y...+ ",
+        "   +++++R.......+++++++ ",
+        "       +RRRR.JWZ+       ",
+        "       ++++++ooo+       ",
+        "                        "
+      ],
+      "palettes": [ "standard_afsshelter_palette" ],
+      "place_item": [ { "item": "television", "repeat": 1, "x": 21, "y": 17 }, { "item": "toaster", "repeat": 1, "x": 6, "y": 17 } ],
```
Comment (actual-nh) [inline data/mods/Aftershock/maps/mapgen/houses/houses.json:36]: Is `repeat: #` always needed? If it's a just-in-case, that's fine, of course.

File: data/mods/Aftershock/maps/mapgen_pallete/houses.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
+[
+  {
+    "type": "palette",
+    "//": "the food should be rotted, but for now the regular food item groups can work for now",
```
Comment (actual-nh) [inline data/mods/Aftershock/maps/mapgen_pallete/houses.json:4]:
```suggestion
    "//": "the food should be rotted, but the regular food item groups can work for now",
```
Yes, I'm compulsive...

File: data/mods/Aftershock/maps/mapgen_pallete/houses.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    },
+    "items": {
+      "t": { "item": "SUS_toilet", "chance": 10, "repeat": [ 1, 3 ] },
+      "D": [ { "item": "wardrobe_unisex_afs", "chance": 100, "repeat": [ 1, 2 ] } ],
+      "O": { "item": "SUS_oven", "chance": 100 },
+      "2": [
+        { "item": "consumer_electronics", "chance": 10, "repeat": [ 1, 3 ] },
+        { "item": "livingroom", "chance": 10, "repeat": [ 1, 3 ] }
+      ],
+      "F": { "item": "SUS_fridge", "chance": 80 },
```
Comment (actual-nh) [inline data/mods/Aftershock/maps/mapgen_pallete/houses.json:66]: Perhaps a freezer should be built into an outside wall, to take advantage of external temperatures?

**Review by actual-nh — COMMENTED** [review]
Nice!

**Review by Maleclypse — APPROVED** [review]

File: data/mods/Aftershock/maps/mapgen/houses/houses.json (hunk 1 above)
Comment (Mom-Bun) [reply data/mods/Aftershock/maps/mapgen/houses/houses.json:36]: Just mostly copy and pasted from the regular house palette, no idea if it is really?

File: data/mods/Aftershock/maps/mapgen_pallete/houses.json (hunk 2 above)
Comment (Mom-Bun) [reply data/mods/Aftershock/maps/mapgen_pallete/houses.json:4]: lol

File: data/mods/Aftershock/maps/mapgen_pallete/houses.json (hunk 3 above)
Comment (Mom-Bun) [reply data/mods/Aftershock/maps/mapgen_pallete/houses.json:66]: Maybe in the future? there is no dedicated freezer furniture afaik right now so it'll have to be added

File: data/mods/Aftershock/maps/mapgen/houses/houses.json (hunk 1 above)
Comment (John-Candlebury) [reply data/mods/Aftershock/maps/mapgen/houses/houses.json:36]: You dont actually need repeat in this case.

**Pull request merged**